.pre-commit-config.yaml

```diff
@@ -0,0 +1,46 @@
+repos:
+#- repo: https://github.com/terrencepreilly/darglint
```

Comment:
Can you delete these commented out sections?

Reply:
done